src/Stepper/Stepper.jsx

```diff
@@ -0,0 +1,306 @@
+import React, {PropTypes} from 'react';
+
+import {getMuiTheme} from '../styles';
+import Paper from '../paper';
+
+const Stepper = React.createClass({
+  propTypes: {
+
+    /**
+     * The current active step index which passed by parent component.
+     */
+    activeStepIndex: PropTypes.number,
+
+    /**
+     * Children should be Step type.
+     */
+    children: PropTypes.node,
+
+    /*
+     * Override inline-style of the content container.
+     */
+    containerStyle: PropTypes.object,
+
+    /**
+     * Create suitable icon for step base on state of the step.
+     */
+    createIcon: PropTypes.func,
```

Comment:
Ok i got it. i will  follow the new standard description.
I don't think `stepIcon` is suitable. Because this is a function, so the name should be verbose.
`getStepIcon` may be better